src/shared/is_node.js

```diff
@@ -12,13 +12,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/* globals module, process */
+/* globals process */
 
-module.exports = function isNodeJS() {
+function isNodeJS() {
```

Comment:
I'm not entirely sure why this *needs* to be a function, since it should be marginally more efficient/simpler to have this be a constant!?

Reply:
I'm fine with making this a constant. That should indeed be simpler given that computing this only once suffices.